aws/resource_aws_imagebuilder_image_pipeline.go

```diff
@@ -127,13 +127,18 @@ func resourceAwsImageBuilderImagePipeline() *schema.Resource {
 				Default:      imagebuilder.PipelineStatusEnabled,
 				ValidateFunc: validation.StringInSlice(imagebuilder.PipelineStatus_Values(), false),
 			},
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
+			"tags_all": tagsSchemaComputed(),
```

Comment:
missed on first review @bflad: just need to update this resource's Read method to set these attributes

Reply:
Updated in https://github.com/hashicorp/terraform-provider-aws/pull/18723/commits/7dc456f1093f7a695dbbb67fa8dcfff3f9712875